volume.go

```diff
@@ -38,6 +39,14 @@ func NewPodVolume(name, path string) *PodVolume {
 	}
 }
 
+// NewPodVolume creates a new PodVolume for file based secrets
+func NewPodVolumeSecret(name, secretPath string) *PodVolume {
```

Comment:
Would we also need/want `PodVolume.Host` in the case of defining a pod volume for file-based secrets, or is that optional?

Reply:
its not even supported. https://mesosphere.github.io/marathon/docs/secrets.html#example-configuration-of-file-based-secrets-for-pod-definitions

The flow is:
1. Define `PodVolume` to provide the secret
2. Define PodVolumeMount us use the secret and specify the container path.

Host is in this case not relevant. Furthermore AFAIK it must not be specified.

Reply:
Gotcha, thanks for explaining.